docs/content/any/project/changelogs/2021-11-30.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Release 2021-11-30
+menuTitle: "2021-11-30"
+any: true
+description: >-
+  Changelog for Release 2021-11-30 (0.24.0) containing new features,
+  bug fixes, and more.
+---
+
+## Oso 0.24.0
+
+### VSCode Extension
```

Comment:
This should be a top-level (h2 / `##`) heading like `## VSCode Extension 0.24.0`